.eslintrc

```diff
@@ -124,7 +124,7 @@
         "use-isnan": 2,
         "valid-jsdoc": 0,
         "valid-typeof": 2,
-        "wrap-iife": 2,
+        "wrap-iife": 0,
```

Comment:
We almost never use iife and the required style change seems to me to be tediously specific, requiring extra `()` that are not required in js, with some other randomly and exceedingly specific requirements for placement of them. Turn the error on, and try and get test/v2-app/index.js to eslint clean and you'll see what I mean. I'd be curious why you think its so useful, but not in this PR, I reignored the apps for  eslint and reverted the iife relaxation.